src/main/kotlin/nl/info/zac/flowable/bpmn/model/ZaaktypeBpmnProcessDefinition.kt

```diff
@@ -38,4 +38,10 @@ class ZaaktypeBpmnProcessDefinition {
     @NotBlank
     @Column(name = "bpmn_process_definition_key")
     lateinit var bpmnProcessDefinitionKey: String
+
+    @Column(name = "zaaktype_omschrijving")
+    lateinit var zaaktypeOmschrijving: String
+
+    @Column(name = "product_aanvraag_type")
+    lateinit var productAanvraagType: String
```

Comment:
I would propose to stick to the same naming as in `ZaakafhandelParameters`:


```suggestion
    @Column(name = "zaaktype_omschrijving")
    lateinit var zaaktypeOmschrijving: String

    @Column(name = "productaanvraagtype")
    lateinit var productaanvraagtype: String
```

And although you mentioned above that `productaanvraagtype` should be non-null I think that not all zaaktypes can be started from OpenFormulieren. So perhaps we still need it as nullable

Reply:
done